src/Migrate.php

```diff
@@ -599,6 +600,35 @@ function (array $v) use ($database, $schemaName, $tableName) {
                     }
                 }
             }
+
+            $this->logger->info(sprintf('Cloning views from database "%s"', $database));
+            $this->sourceConnection->grantRoleToUser($this->config->getSourceSnowflakeUser(), $databaseRole);
+            $this->sourceConnection->useRole($databaseRole);
+            $views = $this->sourceConnection->fetchAll(sprintf(
+                'SHOW VIEWS IN DATABASE %s;',
+                Helper::quoteIdentifier($database),
+            ));
+
+            $views = array_filter($views, fn($v) => !in_array($v['schema_name'], self::SKIP_CLONE_SCHEMAS));
+
+            foreach ($views as $view) {
+                $this->destinationConnection->useRole($view['owner']);
```

Comment:
a tady urcite exisuje ten ROLE?

Reply:
@pivnicek view může založit jen WS role... celá ta struktura uživatelů/rolí by měla být vytvořená

Reply:
parada, diky!